conftest.py

```diff
@@ -0,0 +1,16 @@
+import sys
+
+collect_ignore = []
+if sys.platform != 'win32':
+    collect_ignore += [
+        'dateutil/tzwin.py',
+        'dateutil/tz/win.py',
+    ]
+
```

Comment:
Some of the doctests could be cleaned up by adding e.g.

```

@pytest.fixture(autouse=True)
def add_imports(doctest_namespace):
    doctest_namespace["datetime"] = datetime
    [...]
    doctest_namespace["parser"] = parser
```

Reply:
True that.